repose-aggregator/components/filters/open-api-validator-filter/src/main/resources/META-INF/schema/config/open-api-validator.xsd

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  _=_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_=
+  Repose
+  _-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-
+  Copyright (C) 2010 - 2015 Rackspace US, Inc.
+  _-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+  =_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_=_
+  -->
+
+
+<xs:schema targetNamespace="http://docs.openrepose.org/repose/openapivalidator/v1.0"
+           xmlns="http://docs.openrepose.org/repose/openapivalidator/v1.0"
+           xmlns:html="http://www.w3.org/1999/xhtml"
+           xmlns:xs="http://www.w3.org/2001/XMLSchema"
+           elementFormDefault="qualified">
+
+    <!-- Elements -->
+    <xs:element name="open-api-validator" type="OpenApiValidatorConfig"/>
+
+    <!-- Types -->
+    <xs:complexType name="OpenApiValidatorConfig">
+        <xs:annotation>
+            <xs:documentation>
+                <html:p>
+                    The root config type for the OpenAPI Validator filter configuration file.
+                </html:p>
+            </xs:documentation>
+        </xs:annotation>
+
+        <xs:attribute name="href" type="xs:anyURI" use="required">
```

Comment:
Where is the option for masking?

Reply:
I didn't add one since RBAC is not a part of this story. Masking, at least as it exists right now, would be for `403`s which are returned when there is a role mismatch. For context, the `api-validator` filter schema names the corresponding attribute `mask-rax-roles-403`.